homeassistant/components/switch/enigma.py

```diff
@@ -0,0 +1,136 @@
+"""
+homeassistant.components.switch.enigma
```

Comment:
I would like to suggest you keep using this switch platform as a custom component for now*. I don't want to merge it into Home Assistant. Toggling media players on / off should be done via the media player component. I have just added a turn_on method to the media_player component (10d74bb37e432e36a5b14041c586776ac79822e6) and will look into updating the frontend in a bit.

\*: Move this file to `config/custom_components/switch/enigma.py` and it should work as if it was built-in